drivers/src/hand_off.rs

```diff
@@ -275,10 +275,15 @@ pub struct FirmwareHandoffTable {
     /// Fuse log Address
     pub fuse_log_addr: u32,
 
+    // RtAlias public key.
```

Comment:
nit: rust style for doc comments (function, struct, member comments) is to start with `///`. If we every auto-generate documentation for these libraries, the docs will pick up all the doc comments.

Reply:
Done